src/util/logging.cpp

```diff
@@ -263,8 +258,13 @@ void handleMessage(
         // TODO: Remove the following line.
         // Do not write debug log messages into log file if log level
         // Debug is not enabled starting with release 2.4.0! Until then
-        // write debug messages unconditionally into the log file
-        writeFlags |= WriteFlag::File;
+        // write debug messages into the log file, but skip controller I/O
+        // to avoid flooding the log file.
+        if (!(categoryName.startsWith("controller") &&
```

Comment:
- Add separators to search strings, i.e. `controller.`, `.input`, and `.output`?
- Move code into `inline bool isControllerDebugLoggingCategory(const QString& categoryName)`
- Only evaluate this function which involves a string search if `WriteFlag::File` is not already set

Reply:
> Add separators to search strings, i.e. controller., .input, and .output?

The code already does this.

Reply:
done